### PR TITLE
fix: improved analytics queries and output

### PR DESCRIPTION
**Pull request by Pagebakers**
…ectly

## Motivation and Context

- [ ] Tests or benchmark included
- [ ] Documentation is changed or added on [https://app.gitbook.com/](https://app.gitbook.com/)
- [ ] PR title must follow [conventional-commit-standard](https://github.com/wundergraph/wundergraph/blob/main/CONTRIBUTING.md#conventional-commit-standard)

### CODE REVIEW

**Review by StarpTech — APPROVED** [review]
LGTM

File: controlplane/src/core/repositories/analytics/MetricsDashboardRepository.ts [hunk 1]
```diff
@@ -321,7 +321,7 @@ export class MetricsDashboardRepository {
         top:
           top5Response.data?.result.map((v: any) => ({
             name: v.metric.wg_operation_name || 'unknown',
-            value: v.value[1] || '0',
+            value: !v.value[1] || v.value[1] === 'NaN' ? '0' : v.value[1],
```
Comment (thisisnithin) [inline controlplane/src/core/repositories/analytics/MetricsDashboardRepository.ts:324]: Maybe other places would face the same issue? I saw it with P95 latency too. (the big number in the card) We should have a resolveValue fucntion and apply it to every field here where we access value

**Review by thisisnithin — CHANGES_REQUESTED** [review]

File: controlplane/src/core/repositories/analytics/MetricsDashboardRepository.ts (hunk 1 above)
Comment (Pagebakers) [reply controlplane/src/core/repositories/analytics/MetricsDashboardRepository.ts:332]: Good point

**Review by StarpTech — APPROVED** [review]
LGTM

**Review by thisisnithin — APPROVED** [review]

**Pull request merged**